external/src/main/java/com/redhat/cloud/policies/engine/handlers/TriggersHandler.java

```diff
@@ -739,15 +741,16 @@ public void findGroupMembers(RoutingContext routing) {
             @DocResponse(code = 404, message = "Trigger not found", response = ResponseUtil.ApiError.class),
             @DocResponse(code = 500, message = "Internal server error.", response = ResponseUtil.ApiError.class)
     })
-    @Timed(absolute = true, tags = {"path=/triggers", "method=GET"})
+    @Timed
+            //(absolute = true, tags = {"path=/triggers", "method=GET"})
```

Comment:
Are the old tags automatically added by Micrometer? Should that comment be removed?

Reply:
I tried to somehow add the tags with the last commit, but absolute = true isn't available